app/views/index.scala.html

```diff
@@ -244,6 +255,7 @@
 <script type="text/javascript" src='@routes.Assets.at("javascripts/lib/i18next.min.js")'></script>
 <script type="text/javascript" src='@routes.Assets.at("javascripts/lib/i18nextXHRBackend.min.js")'></script>
 <link href='@routes.Assets.at("stylesheets/choropleth.css")' rel="stylesheet"/>
+<link href='@routes.Assets.at("stylesheets/mlGIF.css")' rel='stylesheet'/>
```

Comment:
We actually have all our CSS for the landing page in `main.css`. That should change at some point, but for now can you move your CSS there instead?